src/showcase/astro.config.ts

```diff
@@ -1,11 +1,12 @@
 import { defineConfig } from "astro/config";
+import path from "path";
```

Comment:
Just in case you want to apply the same pattern as the existing config:

```
import { join } from "path";
```

Reply:
almost did, but then realized it was confusing since there a lots of `join` functions doing different things. WDYT?

Reply:
No opinion, like I said, "just in case".